### PR TITLE
Issue#1175

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
I have made some initial changes which solve most of the issues. I would like to ensure that my approach is correct. Please review and give feedback. You can see the changes by selecting Spanish as I have added translation for the same. #1175

### CODE REVIEW

**Comment by tijlleenders** [comment]
@Tushar-4781 Can you confirm this is the right approach?

File: src/translations/en/translation.json [hunk 1]
```diff
@@ -105,5 +105,46 @@
...
   "Water the plants 🪴 indoors": "Water the plants 🪴 indoors",
-  "Me time 🧘🏽😌": "Me time 🧘🏽😌"
+  "Me time 🧘🏽😌": "Me time 🧘🏽😌",
+  "dontaskagain": "Don't ask again for this action?",
+  "Note": "Note",
+  "Complete goal": "Complete goal",
+  "Delete goal": "Delete goal",
+  "Share goal": "Share goal",
+  "Choose contact": "Choose contact",
+  "Collaborate on goal": "Collaborate on goal",
```
Comment (Tushar-4781) [inline src/translations/en/translation.json:115]: @vinaybadgujar102 Just make these keys one word, like "share goal" => "shareGoal" and follow camel case style. rest looks good to me.

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: src/translations/en/translation.json (hunk 1 above)
Comment (vinaybadgujar102) [reply src/translations/en/translation.json:115]: To make the requested changes I also have to change the values of "getConfirmButtonText" object in "myGoals.ts".

Comment (vinaybadgujar102) [reply src/translations/en/translation.json]: @Tushar-4781 made the required changes

**Comment by vinaybadgujar102** [comment]
@Tushar-4781

File: src/translations/hi/translation.json [hunk 2]
```diff
@@ -105,5 +105,46 @@
...
+    "delete": {
+      "header": "क्या आप इस लक्ष्य को हटाना चाहते हैं?",
+      "note": "इससे सहयोग समाप्त हो जाएगा और आपको कोई अद्यतन प्राप्त नहीं होगा।"
+    },
+    "archive": {
+      "header": "क्या आप इस लक्ष्य को संग्रहित करना चाहते हैं?",
+      "note": "इससे सहयोग समाप्त हो जाएगा और आपको कोई अद्यतन प्राप्त नहीं होगा।"
+    }
+  }
 }
```
Comment (Tushar-4781) [inline src/translations/hi/translation.json:150]: Please add EOL in all reqd files.

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: src/translations/hi/translation.json (hunk 2 above)
Comment (vinaybadgujar102) [reply src/translations/hi/translation.json:150]: I am not able to understand what EOL means. Can you give me a example?

Comment (Tushar-4781) [reply src/translations/hi/translation.json:150]: Please configure your eslint with the project so that these issues can be fixed in one go.

Comment (Tushar-4781) [reply src/translations/hi/translation.json]: nvm i've resolved for now. can't keep PR(s) open for so long :)

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**